File: example/src/main/java/net/dontdrinkandroot/wicket/example/page/HomePage.java
package net.dontdrinkandroot.wicket.example.page;

import org.apache.wicket.model.IModel;
import org.apache.wicket.model.Model;


public class HomePage extends DecoratorPage<Void>
{

	@Override
	protected IModel<String> createPageTitleModel()
	{
		return Model.of("wicket.example - Overview");
	}

	@Override
	protected IModel<String> createPageHeadingModel()
	{
		return null;
	}

	@Override
	protected void onInitialize()
	{
		super.onInitialize();
	}
}

File: example/src/main/kotlin/net/dontdrinkandroot/wicket/example/page/HomePage.kt
package net.dontdrinkandroot.wicket.example.page

import org.apache.wicket.model.Model

class HomePage : DecoratorPage<Void>() {

    override fun createPageTitleModel() = Model("wicket.example - Overview")

    override fun createPageHeadingModel() = Model<String>(null)
}